release-notes/versioned/pulsar-2.11.0.md

```diff
@@ -0,0 +1,643 @@
+- Use 2.8.x/2.9.x/2.10.x doc set instead of version-specific doc set [#17074](https://github.com/apache/pulsar/pull/17074)
+
+> For the comprehensive list of doc-related improvements in 2.11.0, you can go to [GitHub](https://github.com/apache/pulsar/pulls?q=is%3Apr+milestone%3A2.11.0+label%3Adoc+is%3Aclosed). A special thanks to the following contributors (alphabetic Github IDs) who helped add docs for Pulsar 2.11.0.
+> [704237006](https://github.com/704237006), [AlphaWang](https://github.com/AlphaWang), [Anonymitaet](https://github.com/Anonymitaet), [asafm](https://github.com/asafm), [cbornet](https://github.com/cbornet), [codelipenghui](https://github.com/codelipenghui), [coderzc](https://github.com/coderzc), [Demogorgon314](https://github.com/Demogorgon314), [ericsyh](https://github.com/ericsyh), [fantapsody](https://github.com/fantapsody), [futeng](https://github.com/futeng), [hangc0276](https://github.com/hangc0276), [heesung-sn](https://github.com/heesung-sn), [hrsakai](https://github.com/hrsakai), [HQebupt](https://github.com/HQebupt), [horizonzy](https://github.com/horizonzy), [Huanli-Meng](https://github.com/Huanli-Meng), [gaozhangmin](https://github.com/gaozhangmin), [ikilobyte](https://github.com/ikilobyte), [Jason918](https://github.com/Jason918), [komalatammal](https://github.com/komalatammal), [larshp](https://github.com/larshp), [lgxbslgx](https://github.com/lgxbslgx), [lhotari](https://github.com/lhotari), [liangyepianzhou](https://github.com/liangyepianzhou), [liudezhi2098](https://github.com/liudezhi2098), [liuzhuang2017](https://github.com/liuzhuang2017), [Mans2singh](https://github.com/mans2singh), [massakam](https://github.com/massakam), [mattisonchao](https://github.com/mattisonchao), [maxsxu](https://github.com/maxsxu), [mendonk](mendonk), [merlimat](https://github.com/merlimat), [michaeljmarshall](https://github.com/michaeljmarshall), [misselvexu](https://github.com/misselvexu), [momo-jun](https://github.com/momo-jun), [MTwz](https://github.com/MTwz), [nahguam](https://github.com/nahguam),[nicoloboschi](https://github.com/nicoloboschi), [nodece](https://github.com/nodece), [poorbarcode](https://github.com/poorbarcode), [rdhabalia](https://github.com/rdhabalia), [RobertIndie](https://github.com/RobertIndie), [samredai](https://github.com/samredai), [Shawyeok](https://github.com/Shawyeok), [Sherlock113](https://github.com/Sherlock113), [shibd](https://github.com/shibd), [Shoothzj](https://github.com/Shoothzj), [SignorMercurio](https://github.com/SignorMercurio), [Technoboy-](https://github.com/Technoboy-), [tjiuming](https://github.com/tjiuming), [tisonkun](https://github.com/tisonkun), [urfreespace](https://github.com/urfreespace), [wangjialing218](https://github.com/wangjialing218), [wenbingshen](https://github.com/wenbingshen), [youzipi](https://github.com/youzipi), [zwOvO](https://github.com/zwOvO).
```

Comment:
```suggestion
> [0xflotus](https://github.com/0xflotus), [315157973](https://github.com/315157973), [704237006](https://github.com/704237006), [AlphaWang](https://github.com/AlphaWang), [alpreu](https://github.com/alpreu), [AlvaroStream](https://github.com/AlvaroStream), [AnonHxy](https://github.com/AnonHxy),[Anonymitaet](https://github.com/Anonymitaet), [asafm](https://github.com/asafm), [cbornet](https://github.com/cbornet), [codelipenghui](https://github.com/codelipenghui), [coderzc](https://github.com/coderzc), [D-2-Ed](https://github.com/D-2-Ed), [DaveDuggins](https://github.com/DaveDuggins), [Demogorgon314](https://github.com/Demogorgon314), [ericsyh](https://github.com/ericsyh), [fantapsody](https://github.com/fantapsody), [futeng](https://github.com/futeng), [hangc0276](https://github.com/hangc0276), [heesung-sn](https://github.com/heesung-sn), [hrsakai](https://github.com/hrsakai), [HQebupt](https://github.com/HQebupt), [horizonzy](https://github.com/horizonzy), [Huanli-Meng](https://github.com/Huanli-Meng), [gaozhangmin](https://github.com/gaozhangmin), [ikilobyte](https://github.com/ikilobyte), [Jason918](https://github.com/Jason918), [komalatammal](https://github.com/komalatammal), [labuladong](https://github.com/labuladong), [larshp](https://github.com/larshp), [lgxbslgx](https://github.com/lgxbslgx), [lhotari](https://github.com/lhotari), [liangyepianzhou](https://github.com/liangyepianzhou), [lin-zhao](https://github.com/lin-zhao), [liudezhi2098](https://github.com/liudezhi2098), [liuzhuang2017](https://github.com/liuzhuang2017), [Mans2singh](https://github.com/mans2singh), [massakam](https://github.com/massakam), [mattisonchao](https://github.com/mattisonchao), [maxsxu](https://github.com/maxsxu), [mendonk](mendonk), [merlimat](https://github.com/merlimat), [michaeljmarshall](https://github.com/michaeljmarshall), [misselvexu](https://github.com/misselvexu), [momo-jun](https://github.com/momo-jun), [MTwz](https://github.com/MTwz), [nahguam](https://github.com/nahguam),[nicoloboschi](https://github.com/nicoloboschi), [nodece](https://github.com/nodece), [poorbarcode](https://github.com/poorbarcode), [Pradhyumnakashyap](https://github.com/pradhyumnakashyap), [rdhabalia](https://github.com/rdhabalia), [RobertIndie](https://github.com/RobertIndie), [samredai](https://github.com/samredai), [Shawyeok](https://github.com/Shawyeok), [Sherlock113](https://github.com/Sherlock113), [shibd](https://github.com/shibd), [Shoothzj](https://github.com/Shoothzj), [SignorMercurio](https://github.com/SignorMercurio), [Technoboy-](https://github.com/Technoboy-), [tjiuming](https://github.com/tjiuming), [tisonkun](https://github.com/tisonkun), [urfreespace](https://github.com/urfreespace), [wangjialing218](https://github.com/wangjialing218), [wenbingshen](https://github.com/wenbingshen), [youzipi](https://github.com/youzipi), [zwOvO](https://github.com/zwOvO).
```

Adding a few more new names that contributed to the 2.11 docs.

Reply:
updated